src/site/_includes/pattern.njk

```diff
@@ -1,5 +1,5 @@
 ---
-layout: patterns
+layout: pattern-page
```

Comment:
It seems like we don't want the "Was this page helpful" widget.

I think we should actually just comment it out of the post template. Nobody uses it—Kayce said the data it sent back in analytics was too noisy to be useful.

Reply:
Let's not mix Patterns and Post changes here - will submit separately.

Reply:
yep that's fine